### DISCUSSION TITLE
[Beta] Please don't show random people 's activities in feed.

### DISCUSSION THREAD

**Discussion by Chadha93** [community/community#15792] [Other Feature Feedback, Questions, & Ideas]
The beta is kinda good to use but it gets loaded a lot with random people's activities. Keeping it limited and precise would be great.

**Comment by WuJunkai2004** [comment]
Yeah，and sometimes it shows that someone's action randomly but I do not known why. I just like watching some repositorises which instersted me.

**Comment by charnley** [comment]
I get `You're seeing this because you collaborated with X` in my feed. I don't care about this content; If I wanted to see what projects random people are giving a star, I would press Follow on them.

**Comment by MichaelWheldon** [comment]
Please for the love of all that’s good left on the internet don’t add social features to github.

**Reply by MichaelWheldon** [reply]
I’ve just noticed I’m already getting boosted content from businesses I have no interest in or follow in my feed? I already know Seek and Verb exist, I don’t need github reminding me these massive companies are doing stuff on the internet.

Man this is dumb as hell.

**Comment by yitsushi** [comment]
I don't mind getting "You're seeing this because you collaborated with X", but please tell where and on what did I collaborated with X? It can be useful sometimes, for example if it counts how how "deep" is that connection. For example if I had 10 PRs and a 30 comments on a repo, and another user had 4+3, that's reasonable to recommend, I can still say "show less activity", but if I had 1 PR, no comments on a repo that had 300 PRs in the last 6 months, does not matter who else worked on it, I don't think it's reasonable to recommend them. Maybe I just fixed something that's totally out of my interest, but I use given software and given bug was annoying. (eg: not a UI person, but I can fix a button, or how links are generated in a react component)
